doc/source/ingress/ambassador.md

```diff
@@ -2,28 +2,34 @@
 
 Seldon Core works well with [Ambassador](https://www.getambassador.io/), allowing a single ingress to be used to expose ambassador and [running machine learning deployments can then be dynamically exposed](https://kubernetes.io/blog/2018/06/07/dynamic-ingress-in-kubernetes/) through seldon-created ambassador configurations. In this doc we will discuss how your Seldon Deployments are exposed via Ambassador and how you can use both to do various production rollout strategies.
 
+## Installing Ambassador Edge Stack
```

Comment:
I think it would be good to add a section on the Pure OSS stack install of Ambassador and not in the edge stack that this is not open source?

Reply:
@cliveseldon - I've included instructions for installing the OSS Ambassador API Gateway in the latest commit, hopefully the new changes answer your question.